src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -680,10 +683,11 @@ public Object onTraversalSuccess() {
 
                 if (outputModel != null) {
                     final GATKReport report = writeModelReport(goodModel, badModel, USE_ANNOTATIONS);
-                    try (final PrintStream modelReportStream = new PrintStream(outputModel)) {
+                    try (final PrintStream modelReportStream = new PrintStream(outputModel.getOutputStream())) {
+                        if (modelReportStream.checkError()) {
+                            throw new UserException.CouldNotCreateOutputFile(outputModel, "I/O stream error writing to report output");
+                        }
                         report.print(modelReportStream);
```

Comment:
Same problems here -- need to call `checkError()` after `print()`, not before, and need to handle the exception that can be thrown by the `PrintStream` constructor